### DISCUSSION TITLE
Unable to find dark mode settings on gatsby docs website

### DISCUSSION THREAD

**Discussion by AkromDev** [gatsbyjs/gatsby#34299] [Help]
How can I change gatsby docs website to dark theme? I can't find any settings or toggle icon to turn on dark mode.

**Comment by alex-semenyuk** [comment]
This one https://www.gatsbyjs.com/plugins/gatsby-plugin-dark-mode/ I believe what you are looking for

**Reply by AkromDev** [reply]
@alex-semenyuk Thanks but this plugin is for adding dark mode to our own website. My question was about whether gatsby website has dark mode. I wanted to go through some of gatsby guidelines and usually have time at night and light mode strains my eyes.I could find any settings to turn on dark mode on gatsby documentation website.

**Reply by alex-semenyuk** [reply]
Ah. Got it. Then I believe no, but you can create a request on this.
